sympy/concrete/summations.py

```diff
@@ -200,7 +199,7 @@ def doit(self, **hints):
                 else:
                     return self.func(f, *self.limits[n:])
             f = newf
-
+        
```

Comment:
Spaces that will make the code quality test fail.

Reply:
done